events/wc_events/wc_night_elf_events.txt

```diff
@@ -0,0 +1,221 @@
+﻿namespace = WCNELF
+
+WCNELF.1 = { # Bear, Blades, Crane
+	title = WCNELF_1_TITLE
+	desc = WCNELF_1_DESC
+
+	theme = education
+	override_background = {
+		event_background = wilderness_forest
+	}
+
+	right_portrait = {
+		character = root
+		animation = personality_rational
+	}
+
+	option = {
+		name = WCNELF_1_BEAR
+		add_character_modifier = night_elf_bear_markings
+		hidden_effect = {
+			remove_character_modifier = night_elf_blades_markings
+			remove_character_modifier = night_elf_crane_markings
+			remove_character_modifier = night_elf_leaf_markings
+			remove_character_modifier = night_elf_claws_markings
+			remove_character_modifier = night_elf_wings_markings
+			remove_character_modifier = night_elf_serpent_markings
+			remove_character_modifier = night_elf_owl_markings
+			remove_character_modifier = night_elf_shadow_markings
+			remove_character_modifier = night_elf_no_markings
+		}
```

Comment:
You can replace it and all other `hidden_effect`'s with a new scripted effect declared in `Warcraft-Guardians-of-Azeroth-2\common\scripted_effects`.

Call it `remove_facial_markings_effect` and fire in the `immediate` block. Thus you will remove all the markings before assigning a new one in the options.
https://ck3.paradoxwikis.com/Event_modding#Immediate

Reply:
Fixed in 6816d80cd71f4c9415e37ba8ab61aff70cdf76d3.